src/Compilers/CSharp/Portable/Compiler/MethodBodySynthesizer.cs

```diff
@@ -297,6 +299,7 @@ internal static BoundBlock ConstructFieldLikeEventAccessorBody_WinRT(SourceEvent
             BoundCall processHandlerCall = BoundCall.Synthesized(
                 syntax,
                 receiverOpt: getOrCreateCall,
+                initialBindingReceiverIsSubjectToCloning: Binder.TryReceiverIsSubjectToCloning(getOrCreateCall),
```

Comment:
>Binder.TryReceiverIsSubjectToCloning(getOrCreateCall) [](http://example.com/codeflow?start=58&length=53)

ThreeState.Unknown? I think this method is supposed to produce nodes in their lowered form.